compiler/routes.py

```diff
@@ -0,0 +1,128 @@
+"""
+Provides the main API blueprint for compilation.
+
+Notes
+-----
+mypy doesn't have types for flask.Headers.extend, so those lines are
+excluded from type checking.
+
+"""
+
+from typing import Callable, Union, Iterable, Tuple, Optional
+from functools import wraps
+from http import HTTPStatus as status
+from werkzeug.exceptions import Unauthorized, Forbidden, BadRequest
+from werkzeug.wrappers import Response as WkzResponse
+from flask.json import jsonify
+from flask import Blueprint, current_app, redirect, request, g, send_file
+from flask import Response as FlaskResponse
+
+from arxiv.users.auth.decorators import scoped
+from arxiv.users.auth import scopes
+from arxiv.base import logging
+from arxiv.users.domain import Session, Scope
+from arxiv.users.auth import scopes
```

Comment:
This line and the next line duplicate previous imports.

Reply:
Same as above (c1f323d)